Integrations/Elasticsearch_v2/Elasticsearch_v2.py

```diff
@@ -217,6 +308,23 @@ def test_fetch_query(es):
         return None
 
 
+def test_timestamp_format(timestamp):
+    """if is_fetch is ticked and the TIME_METHOD chosen is a type of timestamp - this function checks that
+        the timestamp is in the correct format.
+
+    Args:
+        timestamp(sting): a timestamp string.
+    """
+    timestamp_in_seconds_len = len(str(int(time.time())))
+
+    if TIME_METHOD == 'Timestamp-Seconds' and (len(timestamp) > timestamp_in_seconds_len or not timestamp.isdigit()):
+        return_error(f"The timestamp fetched is not in seconds.\nFetched: {timestamp}")
```

Comment:
Can you try to identify what it is?

Reply:
done (same in milliseconds part obviously)